### DISCUSSION TITLE
Changes not being tracked by Git in 'app' folder

### DISCUSSION THREAD

**Discussion by jameswthompson93** [DevSkillsHQ/discusssions#22] [Help] [answered]
Hi

I've made some changes to the API in the app folder but Git doesn't seem to recognise this changes so I can't push these changes to Git. I followed the instructions found here and they all commands completed successfully as far as I'm aware: https://docs.devskills.co/collections/85-the-interview-process/articles/342-importing-challenge-boilerplate

Is there something that I missed?

Thanks

**Comment by fenske** [comment, marked as answer]
Hi @jameswthompson93. 👋 I'm Anton from Alva. Happy to assist.

The thing is that submitting your solution via git is part of the main objective of this exercise, so it's important you figure it out by yourself.

However, I can give you some direction regarding how you could approach this.

You could clone your coding test repository one more time and copy-paste the changes you made from your existing app folder. 

That way, the freshly cloned repository will identify the changes so you can commit and push them (remember to do it on a new branch as suggested in the README so you can create a Pull Request).

Hope this helps.

Let me know if you have any other questions.

Best Regards,
Anton.
